cron/cron.ts

```diff
@@ -0,0 +1,33 @@
+import cron from 'cron';
+import config from '../utils/config';
+import sentry from '../utils/sentry';
+import activation from './definitions/activation';
+import automation from './definitions/automation';
+import demarchesSimplifiees from './definitions/demarchesSimplifiees';
+import summary from './definitions/summary';
+
+// Desactivate debug log for production as they are a bit too verbose
+if (!config.activateDebug) {
+  console.log('console.debug is not active - thanks to ACTIVATE_DEBUG_LOG env variable');
+  console.debug = () => {};
+}
+
+sentry.initCaptureConsole();
+
+const jobs = activation
+  .concat(automation)
+  .concat(demarchesSimplifiees)
+  .concat(summary);
```

Comment:
I like having all definitions in one place, to check the timing of jobs (to plan releases for example).
I would use one definition file for all jobs